pkg/util/cloudproviders/gce/gce_tags.go

```diff
@@ -46,7 +46,7 @@ func getCachedTags(err error) ([]string, error) {
 		log.Infof("unable to get tags from gce, returning cached tags: %s", err)
 		return gceTags.([]string), nil
 	}
-	return nil, log.Warnf("unable to get tags from gce and cache is empty: %s", err)
+	return nil, log.Debugf("unable to get tags from gce and cache is empty: %s", err)
```

Comment:
I believe `log.Debugf` doesn't return an error like `log.Warnf` does so you can't use it directly here.

What do you think of having a one time warning and otherwise a debug ?